calendar/calendar.yaml

```diff
@@ -25,10 +25,32 @@
 # Current meetings
 ####################################################################################################
 - id: kf032
-  name: Kubeflow Community Call (US + EMEA)
-  date: 08/18/2020
+  name: Kubeflow Community Call
+  date: 04/15/2025
   time: 8:00AM-8:55AM
-  frequency: weekly
+  frequency: bi-weekly
+  video: https://zoom.us/j/99152427566?pwd=S0djc1FPcXkyNUdneHg3UFR3VTcyQT09
+  attendees:
+    - email: kubeflow-discuss@googlegroups.com
+  description:
+    - |
+      Notes & agenda: https://bit.ly/kf-meeting-notes
+
+      Join with Zoom: https://zoom.us/j/99152427566?pwd=S0djc1FPcXkyNUdneHg3UFR3VTcyQT09
+      Meeting ID: 991 5242 7566
+      Passcode: 645928
+
+      Join with Phone (USA): +1 669 900 6833 or +1 646 558 8656
+      International numbers: https://zoom.us/zoomconference?m=Os1EjlUlpb2_XUMaQ6dX1azqMK5CkfWH
+
+      Meeting Host Link: https://zoom.us/s/99152427566
+  organizer: autobot@kubeflow.org
+
+- id: kf043
+  name: Kubeflow CNCF Graduation Call
+  date: 04/22/2025
+  time: 8:00AM-8:55AM
+  frequency: bi-weekly
   video: https://zoom.us/j/99152427566?pwd=S0djc1FPcXkyNUdneHg3UFR3VTcyQT09
   attendees:
     - email: kubeflow-discuss@googlegroups.com
```

Comment:
I cant comment on the line because its not a diff, but are we going to use a separate google doc for the graduation meeting, or use the community doc?

Reply:
I am ok with re-using the same doc, but do you think it is better to create a new one for this call ?

Reply:
Think we should just use the existing graduation checklist doc we have

Reply:
https://docs.google.com/document/d/1UgG4m0dJa55LSV603bWJBVZOq7do1ENCkuE1B7Qk2Kk/edit?tab=t.0

Reply:
Given that we updated the existing doc, let's use it in the calendar invite for now: https://docs.google.com/document/d/1Wdxt1xedAj7qF_Rjmxy1R0NRdfv7UWs-r2PItewxHpE/edit?tab=t.0#heading=h.wxprv3auoc3z